# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1282095711 25200
#      Tue Aug 17 18:41:51 2010 -0700
# Node ID 686e3bc432a29dd3837e0d5ca4e2497e3dfd11e1
# Parent  4a2dabfb078b140b5ef87466d440241965e0ede4
fntests.m: Add extra newline for more readable output during make.

diff --git a/test/ChangeLog b/test/ChangeLog
--- a/test/ChangeLog
+++ b/test/ChangeLog
@@ -1,8 +1,12 @@
+2010-08-17  Rik <octave@nomad.inbox5.com>
+
+	* fntests.m: Add extra newline for more readable output.
+
 2010-07-30  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am: New clean-local target to remove directories 
 	created automatically during testing.
 
 2010-04-28  John W. Eaton  <jwe@octave.org>
 
 	* test_struct.m: Update expected error message text.
diff --git a/test/fntests.m b/test/fntests.m
--- a/test/fntests.m
+++ b/test/fntests.m
@@ -259,17 +259,17 @@ try
     puts ("of Octave as the needed libraries were not present when Octave\n");
     puts ("was built\n");
   endif
 
   report_files_with_no_tests (files_with_tests, files_with_no_tests, ".m");
   report_files_with_no_tests (files_with_tests, files_with_no_tests, ".cc");
 
   puts ("\nPlease help improve Octave by contributing tests for\n");
-  puts ("these files (see the list in the file fntests.log).\n");
+  puts ("these files (see the list in the file fntests.log).\n\n");
 
   fprintf (fid, "\nFiles with no tests:\n\n%s",
 	  list_in_columns (files_with_no_tests, 80));
   fclose (fid);
 
   page_screen_output (pso);
   warning (warn_state.state, "quiet");
 catch
